yrs/src/block.rs

```diff
@@ -781,6 +806,73 @@ impl ItemContent {
     }
 }
 
+impl std::fmt::Display for ID {
```

Comment:
I've implemented Display traits for core components, so that we can print out BlockStore contents in more compacts and comprehensible way.